assets/vsphere_cloud_config.yaml

```diff
@@ -4,9 +4,7 @@ metadata:
   namespace: openshift-cluster-csi-drivers
   name: vsphere-csi-config
 data:
-  cloud.conf: |
-    [Global]
-    cluster-id = "${CLUSTER_ID}"
-    [VirtualCenter "${VCENTER}"]
-    insecure-flag = "true"
-    datacenters = "${DATACENTERS}"
+  cloud.conf: ""
```

Comment:
This will get populated by what's in the ini file (after replacing the strings in it based on infa and vsphere config). Maybe using go templating here would make it more readable? Or maybe there's a good reason to use ini here?

Reply:
I do not fully understand it. How will go templating help? See comment from @jsafrane below - https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/107#discussion_r965872534

Reply:
His suggestion is basically what I had in mind too, it does not have to be go template but it would look nice imho. It's just an idea to make it more readable, something similar to Helm templates but we pass in the values from code and not values.yaml. So the default value for each key could be `""` if it does not break anything and inject some real values only if that's appropriate - that's what @jsafrane meant I believe.